career/roles-in-tech/roles-in-tech.md

```diff
@@ -7,7 +7,11 @@ About 1 hour
 - 30 minutes for the Independent Practice
 
 ### Prerequisites
+<<<<<<< HEAD:onboarding/roles-in-tech.md
```

Comment:
could you please remove these merge conflict lines?

Reply:
@alodahl miss, the changes here is done according to issue #1205. Where the file professionalism.md is beign moved to /career/professionalism.md and in my issue I had to move this file to /onboadring/professionalism.md. Which one is correct?

Reply:
Onboarding